frontend/src/Movie/Index/ProgressBar/MovieIndexProgressBar.js

```diff
@@ -34,11 +34,11 @@ function MovieIndexProgressBar(props) {
   }
 
   if (movieStatus === 'announced') {
-    movieStatus = translate('NotAvailable');
-  }
-
-  if (movieStatus === 'released') {
-    movieStatus = translate('Missing');
+    movieStatus = 'NotAvailable';
```

Comment:
I think this block needs replacing with if(isAvailable) then Missing else Not available. People have different availability settings thus we can't assume one status = missing/not available